packages/leancode_cubit_utils/example/lib/pages/request/request_page.dart

```diff
@@ -1,14 +1,15 @@
 import 'dart:convert';
 
 import 'package:example/http/client.dart';
-import 'package:example/http/status_codes.dart';
+import 'package:example/pages/common.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:http/http.dart' as http;
 import 'package:leancode_cubit_utils/leancode_cubit_utils.dart';
 import 'package:leancode_hooks/leancode_hooks.dart';
 
-class UserRequestCubit extends RequestCubit<http.Response, String, User, int> {
+class UserRequestCubit extends RequestCubit<http.Response, String, User, int>
+    with RequestHandleResult {
   UserRequestCubit(
     this._request,
```

Comment:
This example is quite ambiguous. We create UserRequestCubit, but we also want to allow the possibility to use any request from outside. We should choose one way:

1. I want only the result and, for example to display it: create hook => final cubit = useQuery(() => client.request());
2. I decided to create a dedicated cubit (for example, if I want to create a few instances) and I should encapsulate everything related to this in one place

Currently, we have something between 1. and 2.

Reply:
I've created a dedicated cubit